src/components/BaseButton.vue

```diff
@@ -0,0 +1,31 @@
+<template>
+ <button type="button" :class="size" :style="{'background-color':backgroundColor}" >Trimite
```

Comment:
When you have more than 1 prop, you should format them on new lines
```
<button
   type="button" 
   :class="size" 
   :style="{ 'background-color': backgroundColor }" 
>
   Trimite
</button>
```

Reply:
Also there is no need to use style here, you can add multiple dynamic classes to :class prop, just use object syntax https://vuejs.org/v2/guide/class-and-style.html#Array-Syntax-1 
`:class=[size, { 'background-color': backgroundColor }]`